From: Bhushan Shah <bhush94@gmail.com>
From: Ratchanan Srirattanamet <ratchanan@ubports.com>
Subject: [PATCH] Fix path for halium

Bhushan provides the initial patch. I (Ratchanan) updated it to work with 64-bit
system.

Forwarded: not-needed
Last-Update: 2020-02-06

--- a/hybris.c
+++ b/hybris.c
@@ -27,7 +27,11 @@
 #include <stdlib.h>
 
 #ifndef LIB_DROID_MEDIA_PATH
-#define LIB_DROID_MEDIA_PATH "/usr/libexec/droid-hybris/system/lib/libdroidmedia.so"
+#ifdef __LP64__ // Both AMD64 and AARCH64
+#define LIB_DROID_MEDIA_PATH "/system/lib64/libdroidmedia.so"
+#else
+#define LIB_DROID_MEDIA_PATH "/system/lib/libdroidmedia.so"
+#endif // __LP64__
 #endif
 
 void *android_dlopen(const char *name, int flags);
